### DISCUSSION TITLE
how to make mobile-first website

### DISCUSSION THREAD

**Discussion by rajeshtva** [quasarframework/quasar#10301] [General - Components / Directives / etc]
i am working on website project. which is required to be mobile-first. but when i search _mobile-first_ in official documentation,  then i don't find any mention of this. so my questions are

 - how can i design my website like this?. 
 - do i need to use media query or **$q.is.platform** and how can i use these two?
 - do i need to define seperate block for different kinds of devices for this?

 i am a beginner with medium knowledge of vue framework but poor knowledge of ui/ux and appropriate css.

**Comment by MilosPaunovic** [comment]
Take a closer look at a Quasar spacing classes here:
https://quasar.dev/style/spacing

What you can do is create one layout with different classes for different screen sizes.
